pipenv/utils.py

```diff
@@ -32,6 +32,7 @@
 
 from . import environments
 from .pep508checker import lookup
+from .exceptions import PipenvUsageError
```

Comment:
We are using ```isort``` to sort all the imports, I think ```from .exceptions import PipenvUsageError``` should be placed before ```from .pep508checker import lookup```. So sorry I forget to mention it earlier.